bigquery/google/cloud/bigquery/magics.py

```diff
@@ -434,6 +444,26 @@ def _cell_magic(line, query):
     else:
         max_results = None
 
+    error = None
+
+    if not re.search(r"\s", query.rstrip()):
+        table_id = query.rstrip()
+
+        try:
+            rows = client.list_rows(table_id, max_results=max_results)
+        except Exception as ex:
+            error = str(ex)
+        if error:
```

Comment:
could this be simplified to return from the exception?

Reply:
it definitely can.  fixed in 7a13ca4